### PR TITLE
Move to working sbt plugin repo

### PR DESCRIPTION
**Pull request by bram2000**
the gabrielelana asdf-sbt repo has gone stale and currently
the `list-all` is broken on master. My fork has this fixed
and happy to take over ownership.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ itself) to look for plugins.
 | rlwrap | [asdf-community/asdf-rlwrap](https://github.com/asdf-community/asdf-rlwrap) | [![Main workflow](https://github.com/asdf-community/asdf-rlwrap/workflows/Main%20workflow/badge.svg)](https://github.com/asdf-community/asdf-rlwrap/actions)
 | Ruby      | [asdf-vm/asdf-ruby](https://github.com/asdf-vm/asdf-ruby) | [![Build Status](https://travis-ci.org/asdf-vm/asdf-ruby.svg?branch=master)](https://travis-ci.org/asdf-vm/asdf-ruby)
 | Rust      | [code-lever/asdf-rust](https://github.com/code-lever/asdf-rust) | [![Build Status](https://travis-ci.org/code-lever/asdf-rust.svg?branch=master)](https://travis-ci.org/code-lever/asdf-rust)
-| SBT       | [gabrielelana/asdf-sbt](https://github.com/gabrielelana/asdf-sbt) | [![Build Status](https://travis-ci.org/gabrielelana/asdf-sbt.svg?branch=master)](https://travis-ci.org/gabrielelana/asdf-sbt)
+| SBT       | [bram2000/asdf-sbt](https://github.com/gabrielelana/asdf-sbt) | [![Build Status](https://travis-ci.org/gabrielelana/asdf-sbt.svg?branch=master)](https://travis-ci.org/gabrielelana/asdf-sbt)
```
Comment (imbsky) [inline README.md]:
```suggestion
| SBT       | [bram2000/asdf-sbt](https://github.com/bram2000/asdf-sbt) | [![Build Status](https://travis-ci.org/gabrielelana/asdf-sbt.svg?branch=master)](https://travis-ci.org/gabrielelana/asdf-sbt)
```

Comment (bram2000) [reply README.md]: Oops yes good spot.

**Comment by imbsky** [comment]
Looks good to me.

**Comment by imbsky** [comment]
Thanks @bram2000 for your contribution!

**Pull request merged**